geopandas/io/tests/test_arrow.py

```diff
@@ -328,8 +330,10 @@ def test_to_parquet_fails_on_invalid_engine(tmpdir):
     with pytest.raises(
         ValueError,
         match=(
-            "GeoPandas only supports using pyarrow as the engine for "
-            "to_parquet: 'fastparquet' passed instead."
+            re.escape(
+                "GeoPandas only supports using pyarrow as the engine for "
+                "to_parquet: 'fastparquet' passed instead."
+            )
```

Comment:
```suggestion
            "GeoPandas only supports using pyarrow as the engine for "
            "to_parquet: 'fastparquet' passed instead"
```

It might be that it is only the final dot that triggered the issue?

Reply:
Yes, it is only the final ".". I removed the final "." instead where applicable.